docs/source/index.rst

```diff
@@ -9,7 +9,7 @@ Welcome to Workflow's documentation!
 
 A Python toolkit for building interatomic potential creation and atomistic simulation workflows. 
 
-The main functions of Workflow is to efficiently parallelise operations over a set of atomic configurations (Atomic Simulation Environment's "Atoms" objects). Given an operation that is defined to act a single configuration (e.g. evaluate energy of a structure with CASTEP ASE calculator), Workflow may apply the operation to multiple configurations in parallel. Workflow also interfaces with [ExPyRe](https://github.com/libAtoms/ExPyRe/tree/main/expyre) to manage evaluation of (autoparallelized) python functions via a queueing system on a (remote) cluster. 
+The main functions of Workflow is to efficiently parallelise operations over a set of atomic configurations (Atomic Simulation Environment's "Atoms" objects). Given an operation that is defined to act a single configuration (e.g. evaluate energy of a structure with CASTEP ASE calculator), Workflow may apply the operation to multiple configurations in parallel. Workflow also interfaces with `ExPyRe <https://github.com/libAtoms/ExPyRe/tree/main/expyre>`_ to manage evaluation of (autoparallelized) python functions via a queueing system on a (remote) cluster. 
```

Comment:
`main functions of` -> `main function of`

Given the generic nature of the word "workflow", what do you think about using "wfl" as the default name?

Reply:
> `main functions of -> main function of`

changed, thanks

> Given the generic nature of the word "workflow", what do you think about using "wfl" as the default name?

Do you think just capitalised "Workflow" would work when we're talking about the package?

Reply:
Not really. Maybe all caps.  The capitalization doesn't stand out enough to me.

Reply:
Hmm maybe "WFL" then? But I don't quite like using an abbreviation still...

Reply:
Actually, maybe this would be better discussed during the next meeting?

Reply:
thanks! I'll merge